features/strings.yml

```diff
@@ -0,0 +1,54 @@
+name: String
+description: The String object is used to store and manipulate a sequence of characters.
+spec: https://tc39.es/ecma262/multipage/text-processing.html#sec-string-objects
```

Comment:
See https://github.com/web-platform-dx/web-features/pull/1567#pullrequestreview-2232885174. I think we should add a `group` here and include it with other primitive types.

Reply:
done